src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -185,6 +205,20 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
                     { this.renderGroup(PreferencesUserSettingsTab.TIME_SETTINGS) }
                 </div>
 
+                <div className="mx_SettingsTab_section">
+                    <span className="mx_SettingsTab_subheading">{ _t("Presence") }</span>
+                    <span className="mx_SettingsTab_subsectionText">
+                        { _t("Choose what others can see from your presence.") }
```

Comment:
What about "Share your activity and status with others."?

Reply:
Sounds great!!